cmd/create/cluster/cmd.go

```diff
@@ -721,7 +720,7 @@ func initFlags(cmd *cobra.Command) {
 		&args.billingAccount,
 		"billing-account",
 		"",
-		"Account used for billing subscriptions purchased via the AWS marketplace",
+		"Account used for billing subscriptions purchased via the AWS console for ROSA",
```

Comment:
hello!

sorry to crash the PR but, for some reason, I got email notifs about this content, so here I am. I'm Jocelyn, writer for OCM.

if I were to edit this content, i'd tell you that, for translation reasons, our style guides prohibit us from using "via." "through" is better.

Reply:
Hi @jseseCCS , no apologies needed, thanks for your input.

I will review and address your comments.

Reply:
+1

Reply:
updated: `via` to `through`